code/modules/antagonists/malf_ai/malf_ai.dm

```diff
@@ -159,6 +159,9 @@
 
 	to_chat(malf_ai, "Your radio has been upgraded! Use :t to speak on an encrypted channel with Syndicate Agents!")
 
+	if(malf_ai.malf_picker)
+		stack_trace("Attempted to give malf AI malf picker to \[[owner]\], who already has a malf picker.")
+		return
```

Comment:
You should not stack trace in here, this is like, a valid case? 

I realize the the issue I made a typo, the stack trace should be in `add_malf_picker`. But you still need code in this proc to handle what to do if they already have a malf picker.